ui/plugins/pluginpanel.html

```diff
@@ -206,6 +206,13 @@
           <label>Log parsing status:</label>
           <span>No logs</span>
         </li>
+        <li class="small" ng-click="backfillJob()">
+          <label>
+            <a title="Trigger this jobtype back to the previous time it was run">
+                Backfill this job
+            </a>
+          </label>
+        </li>
```

Comment:
> We've an open bug for this but the number escapes me offhand. It's late here but I'll try to dig it up tomorrow.

Well, we had something like it, but for jobs with multiple logs. Just fyi.
Bug [1113264](https://bugzilla.mozilla.org/show_bug.cgi?id=1113264)